### PR TITLE
chore(version): increase default version; remove pid dir on clean; update pillar.example

### PR DESCRIPTION
**Pull request by noelmcloughlin**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [x] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [x] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

This PR is minor:
- use more modern version as default
- remove /var/run/postgresql if it exists on clean
- update pillar.example version and note on CIDR

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

**Comment by myii** [comment]
@noelmcloughlin Testing this in the CI would mean adjusting this section here:

https://github.com/saltstack-formulas/postgres-formula/blob/1d34f8c56fc8254bab4d175e465915bd97dec0e1/test/salt/pillar/postgres.sls#L17-L22

Then we'll see which platforms can handle the given version number.

File: test/salt/pillar/postgres.sls [hunk 1]
```diff
@@ -18,7 +18,7 @@ postgres:
   {%- if not (grains.os_family == 'Debian') %}
   version: '9.6'
   {%- else %}
-  version: '10'
+  version: '13'
   {%- endif %}
```
Comment (myii) [inline test/salt/pillar/postgres.sls:22]: [Just checked](https://gitlab.com/myii/postgres-formula/-/pipelines/315779405), we no longer need the `if` block:

```suggestion
  version: '13'
```

**Review by myii — CHANGES_REQUESTED** [review]
We can use `version: '13'` for testing across all platforms.

**Comment by myii** [comment]
@vutny Are you OK if `version: 13` is used throughout?

**Comment by noelmcloughlin** [comment]
@myii thanks for the review - I implemented your suggested changes; version 13 should be fine across all OS.

**Review by vutny — APPROVED** [review]
Looks solid, tests are passing. Thanks!

**Review by myii — APPROVED** [review]

**Comment by myii** [comment]
@noelmcloughlin Merged, thanks for the PR.
Appreciate the review, @vutny.

**Pull request merged**